src/Lucene.Net/Store/BufferedIndexInput.cs

```diff
@@ -168,7 +168,7 @@ public override void  ReadBytes(byte[] b, int offset, int len, bool useBuffer, I
 			}
 		}
 		
-		private void  Refill(IState state)
+		public void  Refill(IState state)
```

Comment:
Does it need to be public? Wouldn't `protected` be sufficient?

Reply:
Yeah, done